ui/raidboss/data/06-ew/trial/rubicante.ts

```diff
@@ -0,0 +1,284 @@
+00080004 = Clear CW arrows
+00200010 = Arrows rotating CCW
+00400004 = Clear CCW arrows
+
+04 flags:
+00010001 = cardinals safe?
+00200020 = intercards safe?
+00080004 = clear indicator
+ */
+
+// 7CD4 Ghastly Torch during add phase *is* an aoe but is constant and small, so skipped.
+
+// TO-DO:
+//  - Rewrite Ordeal of Purgation to use Map Effects and identify directional source of cone/safe area(s)
+//  - Flamerake - identify initial direction of line cleave?
+
+export interface Data extends RaidbossData {
+  decOffset?: number;
+  dualfireTarget: boolean;
+}
+
+// First headmarker is Shattering Heat (tankbuster on MT)
+const firstHeadmarker = parseInt('0156', 16);
+const getHeadmarkerId = (data: Data, matches: NetMatches['HeadMarker']) => {
+  if (typeof data.decOffset === 'undefined')
+    data.decOffset = parseInt(matches.id, 16) - firstHeadmarker;
+  return (parseInt(matches.id, 16) - data.decOffset).toString(16).toUpperCase().padStart(4, '0');
+};
+
+const triggerSet: TriggerSet<Data> = {
+  zoneId: ZoneId.MountOrdeals,
+  timelineFile: 'rubicante.txt',
+  initData: () => {
+    return {
+      dualfireTarget: false,
+    };
+  },
+  triggers: [
+    {
+      id: 'Rubicante Headmarker Tracker',
+      type: 'HeadMarker',
+      netRegex: {},
+      condition: (data) => data.decOffset === undefined,
+      run: (data, matches) => {
+        getHeadmarkerId(data, matches);
+      },
+    },
+    {
+      id: 'Rubicante Inferno Raidwide',
+      type: 'StartsUsing',
+      netRegex: { id: '7CEA', source: 'Rubicante', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Rubicante Ordeal of Purgation',
+      type: 'StartsUsing',
+      netRegex: { id: ['7CC4', '80E8'], source: 'Rubicante', capture: false },
+      alertText: (_data, _matches, output) => output.avoid!(),
+      outputStrings: {
+        avoid: {
+          en: 'Avoid cone',
+        },
+      },
+    },
+    {
+      id: 'Rubicante Shattering Heat',
+      type: 'StartsUsing',
+      netRegex: { id: '7CEB', source: 'Rubicante' },
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'Rubicante Blazing Rapture',
+      type: 'StartsUsing',
+      netRegex: { id: '7CD2', source: 'Rubicante', capture: false },
+      // This is a ~14 second cast.
+      delaySeconds: 5,
+      response: Responses.bigAoe(),
+    },
+    {
+      id: 'Rubicante Inferno Spread',
+      type: 'StartsUsing',
+      netRegex: { id: '7CDA', source: 'Rubicante' },
+      condition: Conditions.targetIsYou(),
+      response: Responses.spread(),
+    },
+    {
+      id: 'Rubicante Flamerake',
+      type: 'StartsUsing',
+      netRegex: { id: '7CDD', source: 'Rubicante', capture: false },
+      // Spinning compass animation is 6 seconds; damage goes out 2.6 seconds later
+      delaySeconds: 3,
+      infoText: (_data, _matches, output) => output.avoid!(),
+      outputStrings: {
+        avoid: {
+          en: 'Avoid line cleave, then in',
```

Comment:
I think this could just be `Outputs.in`

Reply:
See note above - I think this part of the same conversation.  We could just do this, but the timing would be super-tight because we'd want it to display once it was safe to move in (after the snapshot for the `7CDF` damage and before the `7CE0` damage, which is only 1.0 seconds apart.